README.md

```diff
@@ -381,7 +381,15 @@ If you're looking for other projects to contribute to please see the
     # checks if var is cool
   end
   ```
+* Private function with the same name of public functions, should start with `do_`.
 
+  ```elixir
+  def sum(list), do: _sum(list, 0)
+  
+  # private functions
+  defp do_sum([], total), do: total
+  defp do_sum([ head | tail ], total), do: _sum(tail, head+total)
```

Comment:
Also, could you use `[head|tail]` (with no spacing) as that's the convention right now (and that's how it's used in the rest of this style guide)?